src/raster/tests.rs

```diff
@@ -693,3 +693,41 @@ fn test_rasterize() {
         [0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 1, 1, 0, 0, 0, 1, 1, 0, 0, 0, 0, 0, 0,]
     );
 }
+
+#[test]
+fn test_vsi_read_dir() {
+    use crate::vsi;
+    use std::collections::HashSet;
+
+    // Concatenate "/vsizip/" prefix.
+    let path = [
+        "/vsizip/",
+        fixture!("test_vsi_read_dir.zip").to_str().unwrap(),
+    ]
+    .concat();
+
+    // Read without recursion.
+    let expected = HashSet::from(["folder", "File 1.txt", "File 2.txt", "File 3.txt"]);
+    let files = vsi::read_dir(path.as_str(), false).unwrap();
+    for file in files {
+        assert!(expected.contains(file.as_str()));
+    }
```

Comment:
You get back a `Vec<String>` so you can just `assert_eq!` once  I guess.

Reply:
I don't think there's any guarantee about the order.